docs/reference/requirement-sets/excel-api-online-requirement-set.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Excel JavaScript API online-only requirement set
+description: 'Details about the ExcelApi Online requirement set'
+ms.date: 11/15/2019
+ms.prod: excel
+localization_priority: Normal
+---
+
+# Excel JavaScript API online-only requirement set
+
+The `ExcelApi Online` requirement set is a special requirement set that includes features that are only available for Excel on the web. APIs in this requirement set are considered to be production APIs (not subject to undocumented behavioral or structural changes) for the Excel on the web host. `ExcelApi Online` are considered to be "preview" APIs for other platforms (Windows, Mac, iOS) and may not be supported by any of those platforms.
+
+APIs in the `ExcelApi Online` requirement set will be moved to a numbered requirement set once they are supported on platforms. The members of the `ExcelApi Online` requirements set will change as features are added across the Excel ecosystem.
```

Comment:
Could we call it main or standard requirement-set of `ExcelApi` instead of numbered.. unless it is a term we use elsewhere.